scim2/scim2_connector_utils.bal

```diff
@@ -101,3 +101,47 @@ function createRequest (json body) returns http:Request {
     request.setJsonPayload(body);
     return request;
 }
+
+function createUpdateBody (string valueType, string newValue) returns json|error {
+    json body = SCIM_PATCH_ADD_BODY;
+    error Error = {};
+
+    if (valueType.equalsIgnoreCase("nickName")) {
```

Comment:
Can we use switch case statement  here? If you cannot use switch case statement pelase can you format it?

Reply:
Don't have switch case in Ballerina. I formatted the if statements.